modules/artifact_registry_iam/variables.tf

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "artifact_registry_repositories" {
+  description = "Artifact registry repositories list to add the IAM policies/bindings"
+  default     = []
+  type        = list(string)
+}
+
+variable "location" {
+  description = "Location of the provided artifact registry repositories"
+  default     = ""
```

Comment:
in general this sub-module is related to artifact registries repos IAM and not to the repos itself.

In my opinion the location doesn't matter because for specifieing IAM you only need the following things:
- Who needs access (group, user, serviceAccount)
- Which resource should he/she/it permitted on (resource, in our case it's the artifact registry repo)
- What shall the permitted resource be able to do? (role / permissions)

I think that no more information is needed to assign IAM-policies.

The ocation is an aatribite that is contained in the artifact registry repository and from my point of view it isn't mandatory.

Reply:
Allright, `project` & `location` variables are set as required now